### ISSUE TITLE
Notifying addon activity

### ISSUE THREAD

**Issue by adrp** [log69/yesscript2#8]
Is it possible to add a console log saying that this addon is actively blocking scripts/altering CSP? Just spent some time debugging Content Security Policy violation on a website with no CSP defined... Saw some CSP warnings in Firefox, but didn't notice the addon is active.

**Comment by log69** [comment]
I believe Firefox logs CSP blocks in the console.

**Comment by adrp** [comment]
That's true, but there's no indication that those CSP blocks are actually coming from the addon. I was pretty confused seeing them and not finding CSP definition in headers nor metatags.

**Comment by log69** [comment]
Currently I have no time resource for the modifications. Maybe in the future.

A

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
